install-binary.sh

```diff
@@ -83,7 +82,7 @@ verifySupported() {
 
 # getDownloadURL checks the latest available version.
 getDownloadURL() {
-  version="$(cat $HELM_PLUGIN_DIR/$PLUGIN_MANIFEST | grep "version" | cut -d '"' -f 2)"
+  version="$(< "$HELM_PLUGIN_DIR/$PLUGIN_MANIFEST" grep "version" | cut -d '"' -f 2)"
```

Comment:
* Fix [SC2086](https://www.shellcheck.net/wiki/SC2086) at line 85
```
 version="$(cat $HELM_PLUGIN_DIR/$PLUGIN_MANIFEST | grep "version" | cut -d '"' -f 2)"
                 ^-- ℹ️ Double quote to prevent globbing and word splitting.
```

Once fixed a new one appeared:
* Fix [SC2002](https://www.shellcheck.net/wiki/SC2002) at line 85:
```
  version="$(cat "$HELM_PLUGIN_DIR/$PLUGIN_MANIFEST" | grep "version" | cut -d '"' -f 2)"
                 ^-- 💄 Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead.
```
Both commands return the same value:
```console
$ HELM_PLUGIN_DIR=/Users/carlosrh/projects/distribution-tooling-for-helm
$ PLUGIN_MANIFEST=plugin.yaml
$ cat $HELM_PLUGIN_DIR/$PLUGIN_MANIFEST
name: "dt"
version: "0.1.0"
usage: "Distribution Tooling for Helm"
description: "Distribution Tooling for Helm"
command: "$HELM_PLUGIN_DIR/bin/dt"
hooks:
  install: "$HELM_PLUGIN_DIR/install-binary.sh"
  update: "$HELM_PLUGIN_DIR/install-binary.sh -u"

$ cat $HELM_PLUGIN_DIR/$PLUGIN_MANIFEST | grep "version" | cut -d '"' -f 2
0.1.0
$ < $HELM_PLUGIN_DIR/$PLUGIN_MANIFEST grep "version" | cut -d '"' -f 2
0.1.0
```